### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: src/tsung/ts_utils.erl [hunk 1]
```diff
@@ -588,7 +588,7 @@ wait_pids(N) ->
         {ok, _Pid, _Res } ->
             wait_pids(N-1)
     after ?TIMEOUT_PARALLEL_SPAWN ->
-            {error, {timout, N}} % N missing answer
```
Comment (guusdk) [inline src/tsung/ts_utils.erl:591]: Does this lead to a functional change?

Comment (kianmeng) [reply src/tsung/ts_utils.erl:591]: Potentially, yes. Should I revert?

I've checked the `develop` branch, it was only used here with no test case written for verify this.

```
$ rg "timout"
src/tsung/ts_utils.erl
591:            {error, {timout, N}} % N missing answer
```

Comment (guusdk) [reply src/tsung/ts_utils.erl:591]: You might even have fixed a bug with this?

> Potentially, yes. Should I revert?

I am not affiliated to this project, so it is not for me to say. Generally, I would advise to split this up in multiple parts: one that has the spelling fixes that are sure to have no side effects, and other parts for each potential functional change.

Comment (kianmeng) [reply src/tsung/ts_utils.erl:591]: We'll let the repo owner to decide on this. Thanks.

Comment (prefiks) [reply src/tsung/ts_utils.erl:591]: This function doesn't seem to be used internally (possibly by some external consumer?), but i think that change is fine.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**